src/blocks/format/html.tsx

```diff
@@ -2,12 +2,11 @@ import { IWPGBlock } from '../../types'
 import * as React from 'react'
 
 class WPGHtmlBlock extends React.Component<IWPGBlock> {
-
   id = Math.floor(Math.random() * 100000)
 
   componentDidMount() {
-    document.querySelectorAll(`[data-script="${this.id}"]`).forEach(script => {
-      (window as any).eval(script.innerHTML)
+    document.querySelectorAll(`[data-script="${this.id}"]`).forEach((script) => {
+      ;(window as any).eval(script.innerHTML)
```

Comment:
uhm. Whats going on here?

Reply:
Confused as well. Prettier keeps putting that semicolon back. Also do you prefer always or avoid for [arrow function parenthesis](https://prettier.io/docs/en/options.html#arrow-function-parentheses)?

Reply:
it can be avoid on function parenthesis.

Reply:
Okay I'll configure it to avoid on my next PR.